opensearch/services/datasets/datasets.php

```diff
@@ -309,7 +338,14 @@ public function getQueryBody($type) {
 				$body .= "?dataset dco:yearOfPublication ?id . ";
 				$body .= "BIND(str(?id) AS ?label) . ";
 				break;
-				
+
+			case "datatypes":
+				$body .= "?dataset a vivo:Dataset . ";
+				$body .= "?dataset  dco:hasDataType ?id . ";
+				$body .= "?id rdfs:label ?label . ";
```

Comment:
Shouldn't this be 

```
$body .= "?id res:label ?l . ";
$body .= "BIND(str(?l) AS ?label) ." ;
```

What you have hear will make ?label a string of the datatype URI.